src/transformers/generation/logits_process.py

```diff
@@ -438,6 +438,22 @@ def __call__(self, input_ids: torch.LongTensor, scores: torch.FloatTensor) -> to
 
 
 def _get_ngrams(ngram_size: int, prev_input_ids: torch.Tensor, num_hypos: int):
+    """
+    Parameters:
+    The key is a generated word and the value is a list of the next words observed.
```

Comment:
This line is not part of the parameters, but part of the return type :)

Reply:
Oops! Thanks :)